libs/src/api/track.js

```diff
@@ -406,19 +406,18 @@ class Tracks extends Base {
   /**
    * Updates an existing track given metadata. This function expects that all associated files
    * such as track content, cover art are already on creator node.
-   * @param {number} trackId id of the track from chain
-   * @param {number} trackOwnerId id of the owner of the track from chain
    * @param {Object} metadata json of the track metadata with all fields, missing fields will error
    */
-  async updateTrack (metadata, newOwnerId = null) {
+  async updateTrack (metadata) {
     this.REQUIRES(Services.CREATOR_NODE)
     this.IS_OBJECT(metadata)
 
-    const ownerId = newOwnerId || this.userStateManager.getCurrentUserId()
+    const ownerId = this.userStateManager.getCurrentUser()
```

Comment:
const owner? or maybe better const ownerId = this.userStateManager.getCurrentUserId() (I think you could do this in a few other places)